data/pages-content/jobs-board.md

```diff
@@ -3,32 +3,340 @@ title: Jobs Board
+- Create high-quality code that is scalable, reliable and reusable to build an everlasting tool
+- Lead projects and initiatives in the Warehouse Simulation team and set the system up for success
+
+
+<h5>Who you are</h5>
+You have a curious mindset and enjoy exploring the intricacies of systems and processes. At the same time, your practical approach to problems allows you to balance between model abstraction, performance, and development effort by focusing on the big picture rather than getting bogged down in the details.
 
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+- You have a degree in Computer Science, Software Engineering, Computational Science or equivalent
+- You are excited about and experienced in simulation (5+ years in software development)
+- …and have outstanding Python skills!
+- You have a good understanding of relational databases and SQL
+- You know when to use NoSQL solutions like MongoDB, ElasticSearch, and RabbitMQ
+- You know your way around software versioning (git), test suites, technical documentation, and continuous integration and deployment
+- ...and you don't mind collecting bonus points with your cloud and Kubernetes skills
 
+<h5>Technologies you'll use</h5>
+
+- Python 3.9/3.10/3.11
+- SimPy
+- Many Python libraries across various domains
+- PostgreSQL, MongoDB, Snowflake
+- RabbitMQ
+- AWS, Docker, Kubernetes, TeamCity
+- Microservices
+
+</details>
+
+[Apply here](https://picnic.app/careers/jobs/5005708/technology--amp--engineering/amsterdam-north-holland-netherlands/senior-warehouse-simulation-software-engineer)
+
+
+## [The Mobility House GmbH](https://www.mobilityhouse.com/)
+We are a leading provider of smart charging solutions with many years of experience in managing battery storage in energy and power markets, we will work with you to unlock the potential that lies in innovative electric vehicle energy products such as V2G (vehicle to grid).
+
+<details>
+  <summary><h4 id="mobility-house1-teamlead-technology-vgi">Teamlead Technology - VGI</h4></summary>
+  - You lead a team of talented SW engineers. Your team's focus is on developing full-stack solutions to control and interlink Electric Vehicle Charging Processes. 
+  - You design, plan and implement information and infrastructure systems to support business strategies by interacting with assets as well as users.
+  - You are supervising, mentoring & scaling the team, putting emphasis on security, stability and scalability as well as on quality measures such as TDD.
+  - You interact with peers and cross-functional colleagues to fullfil our joint mission towards #NorthStar
 </details>
 
-[Apply here](http://example.com) or contact John Doe: example@example.com
+[Apply here](https://www.mobilityhouse.com/de_de/karriere) or contact Maren Lueess: maren.lueess@moblityhouse.com
```

Comment:
I think, we can direct to the English web page here: https://www.mobilityhouse.com/int_en/career/teamlead-technology-vgi-m-f-d-338.html and point to the specific position

Reply:
fixed! Thanks for going the extra mile to check! And realised their name is Lüeße and not Lueess :D